---
title: "**Seasonal Dynamics of Epiphytic Microbial Communities on Marine Macrophyte Surfaces**"
output:
  pdf_document: 
    keep_tex: yes
    includes:
      in_header: header.tex
fontsize: 12pt
geometry: margin=1.0in
csl: citation_style.csl #Get themes at https://github.com/citation-style-language/styles
bibliography: references.bib
---

```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE}
opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)
opts_chunk$set("message" = FALSE)

inline_hook <- function(x){
	print(x)

	if(is.list(x)){
		x <- unlist(x)
	}

	if(is.numeric(x)){
		if(abs(x - round(x)) < .Machine$double.eps^0.5){
			paste(format(x, big.mark=',', digits=0, scientific=FALSE))
		} else {
			paste(format(x, big.mark=',', digits=1, nsmall=1, scientific=FALSE))
		}
	} else {
    	paste(x)      
	}
}
knitr::knit_hooks$set(inline=inline_hook)
```

\vspace{20mm}
Marino Korlević^1$*$^, Marsej Markovski^1^, Zihao Zhao^2^, Gerhard J. Herndl^2,3^, Mirjana Najdek^1^

1\. Center for Marine Research, Ruđer Bošković Institute, Croatia

2\. Department of Functional and Evolutionary Ecology, University of Vienna, Austria

3\. Department of Marine Microbiology and Biogeochemistry, Royal Netherlands Institute for Sea Research (NIOZ), Utrecht University, The Netherlands

^$*$^To whom correspondence should be addressed:

Marino Korlević

G. Paliaga 5, 52210 Rovinj, Croatia

Tel.: +385 52 804 768

Fax: +385 52 804 780

e-mail: marino.korlevic@irb.hr

Running title: Seasonal dynamics of epiphytic communities

\newpage
\linenumbers
\sisetup{mode=text}
\setlength\parindent{24pt}
\doublespacing

## Abstract
Surfaces of marine macrophytes are inhabited by diverse microbial communities. Most studies focusing on epiphytic communities of macrophytes did not take into account temporal changes or applied low sampling frequency approaches. The seasonal dynamics of epiphytic microbial communities was determined in a meadow of *Cymodocea nodosa* invaded by *Caulerpa cylindracea* and in a monospecific settlement of *Caulerpa cylindracea* at monthly intervals. For comparison the ambient prokaryotic picoplankton community was also characterized. At the OTU level, the microbial community composition differed between the ambient water and the epiphytic communities exhibiting host-specificity. Also, successional changes were observed connected to the macrophyte growth cycle. Taxonomic analysis, however, showed similar high rank taxa (phyla and classes) in the ambient water and the epiphytic communities, with the exception of *Desulfobacterota*, which were only found on *Caulerpa cylindracea*. *Cyanobacteria* showed seasonal changes while other high rank taxa were present throughout the year. In months of high *Cyanobacteria* presence the majority of cyanobacterial sequences were classified as *Pleurocapsa*. Phylogenetic groups present throughout the year (e.g. *Saprospiraceae*, *Rhodobacteraceae*, members without known relatives within *Gammaproteobacteria*, *Desulfatitalea* and members without known relatives within *Desulfocapsaceae*) constituted most of the sequences, while less abundant taxa showed seasonal patterns connected to the macrophyte growth cycle. Taken together, epiphytic microbial communities of the seagrass *Cymodocea nodosa* and the macroalga *Caulerpa cylindracea* appear to be host-specific and contain taxa that undergo successional changes.

\newpage
## Introduction
Marine macrophytes (seagrasses and macroalgae) are important ecosystem engineers forming close associations with microorganisms belonging to all three domains of life [@Tarquinio2019; @Egan2013]. Microbes can live within macrophyte tissue as endophytes or form epiphytic communities on surfaces of leaves, roots, rhizomes and thalli [@Tarquinio2019; @Egan2013; @Hollants2013; @Aires2015]. Epiphytic and endophytic microbial communities exhibit a close functional relationship with the macrophyte host. It has been proposed that this close relationship constitutes a holobiont, an integrated community where the macrophyte organism and its symbiotic partners support each other [@Margulis1991]. In addition, as suggested by the hologenome theory endophytic microbes play a critical role in the adaptation and evolution of the host species [@Aires2015].

Biofilms of microbial epiphytes can contain diverse taxonomic groups and harbour cell abundances from 10^2^ to 10^7^ \si{\cells\per\cm\squared} [@Armstrong2000; @Bengtsson2010; @Burke2011a]. In such an environment a number of positive and negative interactions between the macrophyte and the colonizing microorganisms have been described [@Tarquinio2019; @Egan2013; @Hollants2013]. Macrophytes can promote growth of associated microbes by nutrient exudation [@Wood1981], while in return microorganisms may support macrophyte performance through improved nutrient availability [@Nielsen2001; @deOliveira2012], phytohormone production [@Celdran2012; @Matsuo2003] and protection from toxic compounds [@Kusel2006], oxidative stress [@Sanchez-Amat2010], biofouling organisms [@Dobretsov2002] and pathogens [@Penesyan2009]. Besides these positive interactions, macrophytes can  negatively impact the associated microbes by producing reactive oxygen species [@Weinberger2007] and secondary metabolites [@Saha2011].

All these ecological roles are carried out by a taxonomically diverse community of microorganisms. At higher taxonomic ranks (phyla and classes) microbial taxa, such as *Alphaproteobacteria*, *Gammaproteobacteria*, *Bacteroidota* and *Cyanobacteria*, have been associated with surfaces of seagrass leaves and macroalgal thalli [@Lachnit2011; @Crump2008; @Tujula2010; @Egan2013; @Tarquinio2019; @Ugarelli2019]. While similar high rank taxa have been found on surfaces of different macrophyte species, in order to describe new ecological patterns it is also necessary to focus on lower taxonomic ranks (genus and OTUs) which tend to be host-specific [@Roth-Schulze2016; @Lachnit2011; @Hollants2013]. While the microbial community composition can vary between host species, metagenomic analyses revealed that the majority of microbial functions are conserved, showing that different epiphytic microbial species could be functionally similar [@Burke2011; @Roth-Schulze2016; @Cucio2018]. This discrepancy between the microbial taxonomic and functional composition might be explained by the lottery hypothesis [@Sale1976]. It postulates that an initial random colonization step takes place from a set of functionally equivalent taxonomic groups resulting in taxonomically different epiphytic communities sharing a core set of functional genes [@Burke2011; @Roth-Schulze2016; @Schmidt2015a; @Stratil2013].

Seagrasses are know to form close relationships with microbial communities associated with the surfaces of leaves, roots and rhizomes [@Cucio2016; @Ugarelli2019; @Wang2020; @Ettinger2020; @Crump2018]. For different seagrass species a distinct microbial community from ambient seawater or bulk sediment has been reported, however no species specific communities have been found [@Cucio2016; @Ugarelli2019; @Crump2018]. It seems that seagrasses are selecting the associated microbial community but these microbes have not coevolved with their seagrass plant host. Similar to seagrasses, siphonous macroalgae of the genus *Caulerpa* are also closely associated with their microbial communities [@Aires2015; @Aires2013; @Morrissey2019; @Rizzo2016a; @Stabili2017]. While some studies have found similar culturable bacterial groups associated with the surface of a *Caulerpa* species from different geographic locations [@Stabili2017], other have reported large compositional differences that were mainly attributed to different host species of this genus, biogeography and nutrient levels [@Morrissey2019] again raising the question to which extent are associated communities host-specific.

Since marine macrophytes in temperate zones are exhibiting seasonal changes in growth and physiology [@Agostini2003; @Najdek2020a] it is important to verify if and how surface associated microbial communities are affected by these changes. The majority of studies describing macrophyte epiphytic microbial communities have not included possible seasonal changes [@Ugarelli2019; @Lachnit2009; @Crump2008; @Burke2011a; @Roth-Schulze2016]. If seasonal changes have been taken into account, low temporal frequency, applied methodologies and/or limited number of analysed host species did not allow for a detailed taxonomic analysis [@Tujula2010; @Lachnit2011; @Miranda2013; @Michelou2013; @Mancuso2016; @Bengtsson2012]. In the present study we performed a descriptive analysis of seasonal bacterial and archaeal community dynamics on the surfaces of the seagrass *Cymodocea nodosa*, an abundant seagrass species in the Mediterranean [@Short2001], and siphonous macroalga *Caulerpa cylindracea*, one of the most invasive macroalgal species [@Klein2008; @Boudouresque2009]. Bacterial and archaeal epiphytes were sampled in a meadow of *C. nodosa* invaded by the invasive *C. cylindracea* and in a locality of only *C. cylindracea* located in the proximity of the seagrass meadow. For comparison, the microbial community of the ambient seawater was also characterized. The presence of both macrophytes in the same area enabled (i) the assessment of differences in the bacterial and archaeal communities between host species and settlements of *C. cylindracea* and (ii) the evaluation of differences between surface associated and free living (ambient seawater) communities. In addition, these differences were evaluated on a monthly scale providing insight into seasonal changes (iii).

\newpage
## Materials and methods
### Sampling
Sampling was performed in the Bay of Funtana, northern Adriatic Sea (\ang{45;10;39} N, \ang{13;35;42} E). The sea floor in the bay is partly covered by the invasive macroalga *C. cylindracea* that can be found in a monospecific settlement or mixed with the seagrass *C. nodosa* (\autoref{map}). *C. nodosa* leaves were retrieved from a meadow of *C. nodosa* invaded by the invasive *C. cylindracea* (mixed settlement; depth, 2 -- 2.5 \si{\m}), while *C. cylindracea* thalli were sampled in the same invaded meadow (mixed settlement; depth, 2 -- 2.5 \si{\m}) and in a monospecific settlement (depth, 1 -- 1.5 \si{\m}) of *C. cylindracea* located in the proximity (20 -- 50 \si{\m}) of the invaded meadow at approximately monthly intervals from November 2017 to October 2018 (\autoref{supp-nseq_notus}). Leaves and thalli were collected by diving and transported to the laboratory in containers placed on ice and filled with seawater collected at the sampling site. Upon arrival to the laboratory, *C. nodosa* leaves were cut into sections of 1 -- 2 \si{\cm}, while *C. cylindracea* thalli were cut into 5 -- 8 \si{\cm} long sections. Leaves and thalli were washed three times with sterile artificial seawater (ASW) to remove loosely attached microbial cells. Ambient seawater was collected in 10 \si{\l} containers by diving and transported to the laboratory where 10 -- 20 \si{\l} were filtered through a 20 \si{\um} net. The filtrate was further sequentially filtered through 3 \si{\um} and 0.2 \si{\um} polycarbonate membrane filters (Whatman, United Kingdom) using a peristaltic pump. Filters were briefly dried at room temperature and stored at \num{-80} \si{\degreeCelsius}. Seawater samples were also collected approximately monthly from July 2017 to October 2018.

### DNA isolation
DNA from surfaces of *C. nodosa* and *C. cylindracea* was isolated from a pool of leaves (1 g wet weight) or thalli (2 g wet weight) on the sampling day using a previously modified and adapted protocol that allows for a selective epiphytic DNA isolation [@Korlevic2021; @Massana1997]. Briefly, leaves and thalli were incubated in a lysis buffer and treated with lysozyme and proteinase K. Following the incubations, the mixture containing lysed epiphytic cells was separated from the leaves and thalli and extracted using phenol-chloroform. Finally, the extracted DNA was precipitated using isopropanol. DNA from seawater picoplankton was extracted from 0.2 \si{\um} polycarbonate filters according to @Massana1997 with a slight modification. Following the phenol-chloroform extraction, 1/10 of 3 \si{\Molar} sodium acetate (pH 5.2) was added. DNA was precipitated by adding 1 volume of chilled isopropanol, incubating the mixtures overnight at \num{-20} \si{\degreeCelsius} and centrifuging at 20,000 × *g* and 4 \si{\degreeCelsius} for 20 \si{\minute}. The pellet was washed twice with 500 \si{\ul} of chilled 70 \si{\percent} ethanol and centrifuged after each washing step at 20,000 × *g* and 4 \si{\degreeCelsius} for 5 \si{\minute}. Dried pellets were re-suspended in 50 -- 100 \si{\ul} of deionized water. One DNA sample originating from seawater picoplankton was obtained per each sampling point. 

### Illumina 16S rRNA sequencing
Illumina MiSeq sequencing of the V4 region of the 16S rRNA gene was performed as described previously [@Korlevic2021]. The V4 region of the 16S rRNA gene was amplified using a two-step PCR procedure. In the first PCR, the 515F ($5'$-GTGYCAGCMGCCGCGGTAA-$3'$) and 806R ($5'$-GGACTACNVGGGTWTCTAAT-$3'$) primers from the Earth Microbiome Project (https://earthmicrobiome.org/protocols-and-standards/16s/) were used [@Caporaso2012; @Apprill2015; @Parada2016]. These primers contained on their $5'$ end a tagged sequence. Purified PCR products were sent for Illumina MiSeq sequencing at IMGM Laboratories, Martinsried, Germany. Prior to sequencing at IMGM, the second PCR amplification of the two-step PCR procedure was performed using primers targeting the tagged region incorporated in the first PCR. In addition, these primers contained adapter and sample-specific index sequences. Beside samples, a positive and negative control for each sequencing batch was sequenced. The negative control comprised PCR reactions without DNA template, while for a positive control a mock community composed of evenly mixed DNA material originating from 20 bacterial strains (ATCC MSA-1002, ATCC, USA) was used. Sequences obtained in this study have been deposited in the European Nucleotide Archive (ENA) at EMBL-EBI under the accession number PRJEB37267 (https://www.ebi.ac.uk/ena/browser/view/PRJEB37267).

### Sequence and data analysis
```{r community}
# Loading input data containing sequence abundances and subsequent input data customization
community <- read_tsv("data/mothur/raw.trim.contigs.good.unique.good.filter.unique.precluster.pick.nr_v138.wang.tax.summary") %>%
  filter(!str_detect(taxon, "^Eukaryota")) %>%
  filter(taxon!="Root")

# Remove chloroplast and mitochondrial sequences and subtract their number from higher taxonomic levels to which
# they belong
chloroplast <- filter(community, str_detect(taxon, "^Chloroplast$"))$rankID
mitochondria <- filter(community, str_detect(taxon, "^Mitochondria$"))$rankID

community <- mutate_at(community, 5:ncol(community), list(~case_when(
    rankID==str_extract(chloroplast, "(\\d+\\.){3}\\d+") ~ . - .[taxon=="Chloroplast"],
    rankID==str_extract(chloroplast, "(\\d+\\.){2}\\d+") ~ . - .[taxon=="Chloroplast"],
    rankID==str_extract(chloroplast, "(\\d+\\.){1}\\d+") ~ . - .[taxon=="Chloroplast"],
  TRUE ~ .))) %>%
  filter(!str_detect(taxon, "^Chloroplast")) %>%
  mutate_at(5:ncol(.), list(~case_when(
    rankID==str_extract(mitochondria, "(\\d+\\.){4}\\d+") ~ . - .[taxon=="Mitochondria"],
    rankID==str_extract(mitochondria, "(\\d+\\.){3}\\d+") ~ . - .[taxon=="Mitochondria"],
    rankID==str_extract(mitochondria, "(\\d+\\.){2}\\d+") ~ . - .[taxon=="Mitochondria"],
    rankID==str_extract(mitochondria, "(\\d+\\.){1}\\d+") ~ . - .[taxon=="Mitochondria"],
    TRUE ~ .))) %>%
  filter(!str_detect(taxon, "^Mitochondria")) %>%
  select(-ATCC_1, -ATCC_5, -NC_1)

# Calculating the number of sequences in millions
million_sequences <- filter(community, taxon=="Archaea" | taxon=="Bacteria") %>%
  select(6:ncol(.)) %>%
  sum()/1e6

# Loading metadata
metadata <- read_tsv("data/raw/metadata.csv")

# Calculating number of samples
samples_epiphytes <- filter(metadata, !is.na(station) & station!="F") %>%
  summarise(n=n())
samples_seawater <- filter(metadata, !is.na(station) & station=="F") %>%
  summarise(n=n())

# Calculating the minimal and maximal number of sequences
min_max <- filter(community, taxon=="Archaea" | taxon=="Bacteria") %>%
  summarise_at(6:ncol(.), sum) %>%
  summarise(min=min(.), max=max(.))

# Calculating the relative abundance of Archaea and Bacteria
archaea_bacteria <- filter(community, taxon=="Archaea" | taxon=="Bacteria") %>%
  mutate_at(5:ncol(.), list(~. / sum(.) * 100)) %>%
  gather(key="Group", value="abundance", 6:ncol(.)) %>%
  group_by(taxon) %>%
  summarise(mean=mean(abundance), sd=sd(abundance))
```

```{r shared}
# Loading OTU/sample data
shared <- read_tsv("data/mothur/raw.trim.contigs.good.unique.good.filter.unique.precluster.pick.pick.pick.opti_mcc.shared")

# Calculating the number of OTUs before rarefaction
no_otus <- select(shared, starts_with("Otu")) %>%
  colnames() %>%
  length()

# Loading rarefied community data created during the generation of the richness and diversity calculators plot
load(file="results/numerical/rarefied.Rdata")

# Calculating the number of OTUs after rarefaction
no_otus_rarefied <- select(rarefied, starts_with("Otu")) %>%
  colnames() %>%
  length()

# Calculating the maximal and minimal number of OTUs after rarefaction
otu_min_max <- select(rarefied, starts_with("Otu")) %>%
  decostand(., method="pa") %>%
  transmute(sum=rowSums(.))
```

```{r error}
# Calculation of the error rate based on the ATCC MSA-1002 mock community
error <- read_tsv("data/mothur/raw.trim.contigs.good.unique.good.filter.unique.precluster.pick.pick.pick.error.summary", na="") %>%
  filter(numparents==1) %>%
  mutate(mismatches_weight=weight*mismatches) %>%
  mutate(total_weight=weight*total)
error <- sum(error$mismatches_weight)/sum(error$total_weight)*100

# Calculation of the number of sequences in the negative control
nc <- read_tsv("data/mothur/raw.trim.contigs.good.unique.good.filter.unique.precluster.pick.nr_v138.wang.tax.summary") %>%
  filter(taxon=="Root") %>%
  select(NC_1)
```

Obtained sequences were analysed on the computer cluster Isabella (University Computing Center, University of Zagreb) using mothur (version 1.43.0) [@Schloss2009] according to the MiSeq Standard Operating Procedure (MiSeq SOP; https://mothur.org/wiki/MiSeq_SOP) [@Kozich2013] and recommendations provided by the Riffomonas project to enhance data reproducibility (http://www.riffomonas.org/). For alignment and classification of sequences the SILVA SSU Ref NR 99 database (release 138; http://www.arb-silva.de) was used [@Quast2013; @Yilmaz2014]. Sequences were clustered into operational taxonomic units (OTUs) at a similarity level of 97 \si{\percent}.

Pipeline data processing and visualization was done using R (version 3.6.0) [@RCoreTeam2019] combined with packages vegan (version 2.5-6) [@Oksanen2019], tidyverse (version 1.3.0) [@Wickham2017; @Wickham2019] and multiple other packages [@Xie2014; @Xie2015; @Xie2019; @Xie2019a; @Xie2019b; @Allaire2019; @Xie2018; @Zhu2019; @Neuwirth2014; @Wilke2019]. Observed number of OTUs, Chao1, ACE, exponential of the Shannon diversity index and Inverse Simpson index were calculated after normalization to the minimum number of reads per sample using vegan's function `rrarefy` to account for different sequencing depths [@Oksanen2019]. Chao1 and ACE estimators were calculated using vegan's function `estimateR`, while Shannon and Inverse Simpson diversity indices were retrieved using vegan's function `diversity` [@Oksanen2019]. To express both diversity indices in terms of effective number of OTUs the exponential of the Shannon diversity index was calculated [@Jost2006]. The proportions of shared OTUs and communities between samples and community types (seawater, *C. nodosa* [mixed], *C. cylindracea* [mixed] and *C. cylindracea* [monospecific]) were expressed as the Jaccard's (on presence/absence data) and Bray-Curtis similarity coefficient, respectively. The coefficients were calculated on the OTU data table using vegan's function `vegdist` and converted from dissimilarities to similarities [@Oksanen2019; @Borcard2011; @Legendre2012]. The Principal Coordinates Analysis (PCoA) was performed on Bray-Curtis dissimilarities based on OTU abundances using the function `cmdscale` [@Borcard2011; @Legendre2012]. Differences between communities were tested by performing the Analysis of Similarities (ANOSIM) using the vegan's function `anosim` and 1000 permutations [@Oksanen2019], while differences in relative contributions or proportions of shared OTUs and communities were tested by applying the Mann–Whitney *U* test using the function `wilcox.test`. In addition, differences between community type estimators or indices were tested by performing the Kruskal-Wallis *H* test (function `kruskal.test`) followed by a pairwise comparison using the Mann-Whitney *U* test (function `pairwise.wilcox`). Bonferroni correction was used to address the problem of multiple comparisons.

A total of `r million_sequences` million sequences after quality curation and exclusion of sequences without known relatives (no relative sequences) and eukaryotic, chloroplast and mitochondrial sequences were obtained (\autoref{supp-nseq_notus}). The number of reads per sample ranged between `r min_max$min` and `r min_max$max` sequences (\autoref{supp-nseq_notus}). Even when the highest sequencing effort was applied the rarefaction curves did not level off as commonly observed in high-throughput 16S rRNA amplicon sequencing approaches (\autoref{supp-rarefaction}). Following quality curation and exclusion of sequences as mentioned above reads were clustered into `r no_otus` different OTUs. Read numbers were normalized to the minimum number of sequences (`r min_max$min`, \autoref{supp-nseq_notus}) using previously mentioned vegan's function `rrarefy` resulting in `r no_otus_rarefied` different OTUs ranging from `r min(otu_min_max)` to `r max(otu_min_max)` OTUs per sample (\autoref{supp-calculators}). Based on the ATCC MSA-1002 mock community included in the analysis an average sequencing error rate of `r error` \si{\percent} was determined, which is in line with previously reported values for next-generation sequencing data [@Kozich2013; @Schloss2016]. In addition, the negative controls processed together with the samples yielded only `r nc` sequences after sequence quality curation. The detailed analysis procedure including the R Markdown file is available as a GitHub repository (https://github.com/MicrobesRovinj/Korlevic_EpiphyticDynamics_FrontMicrobiol_2021). 

## Results

```{r calculators}
# Loading rarefied community data created during the generation of the richness and diversity calculators plot
load(file="results/numerical/rarefied.Rdata")

# Copying the sample labels to the rows (input for library vegan)
rarefied <- rarefied %>%
  column_to_rownames("Group")

# Calculating Chao1 and ACE species estimators
estimators <- estimateR(rarefied)
estimators <- as_tibble(t(estimators), .name_repair="unique") %>%
  add_column("Group"=colnames(estimators), .before=TRUE)

# Calculating diversity indices
shannon <- diversity(rarefied, index="shannon")
shannon <- tibble(Group=names(shannon), shannon)
invsimpson <- diversity(rarefied, index="invsimpson")
invsimpson <- tibble(Group=names(invsimpson), invsimpson)

# Transforming the Shannon entropy to the effective number of species
# (http://www.loujost.com/Statistics%20and%20Physics/Diversity%20and%20Similarity/EffectiveNumberOfSpecies.htm)
shannon <- mutate(shannon, shannon=exp(shannon)) %>%
  rename(eshannon=shannon)

# Joining together estimators and indices
estimators_indices <- inner_join(estimators, shannon, by="Group") %>%
  inner_join(., invsimpson, by="Group")

# Loading metadata 
metadata <- read_tsv("data/raw/metadata.csv")

# Joining metadata with estimators and indices
estimators_indices_metadata <- inner_join(metadata, estimators_indices, by=c("ID"="Group")) %>%
  mutate(date=as.Date(date, "%d.%m.%Y"))

# Calculating the mean and standard deviation for each environment
S.obs_f <- filter(estimators_indices_metadata, station=="F") %>%
  summarise(mean=mean(S.obs), sd=sd(S.obs))
S.obs_fcym <- filter(estimators_indices_metadata, station=="FCyM") %>%
  summarise(mean=mean(S.obs), sd=sd(S.obs))
S.obs_fcam <- filter(estimators_indices_metadata, station=="FCaM") %>%
  summarise(mean=mean(S.obs), sd=sd(S.obs))
S.obs_fca <- filter(estimators_indices_metadata, station=="FCa") %>%
  summarise(mean=mean(S.obs), sd=sd(S.obs))

# Calculating the mean and standard deviation for each environment and season
S.obs_season_f <- filter(estimators_indices_metadata, station=="F") %>%
  group_by(season) %>%
  summarise(mean_S.obs=mean(S.obs), sd_S.obs=sd(S.obs),
            mean_S.chao1=mean(S.chao1), sd_S.chao1=sd(S.chao1),
            mean_S.ACE=mean(S.ACE), sd_S.ACE=sd(S.ACE),
            mean_eshannon=mean(eshannon), sd_eshannon=sd(eshannon),
            mean_invsimpson=mean(invsimpson), sd_invsimpson=sd(invsimpson))
S.obs_season_fcym <- filter(estimators_indices_metadata, station=="FCyM") %>%
  group_by(season) %>%
  summarise(mean_S.obs=mean(S.obs), sd_S.obs=sd(S.obs),
            mean_S.chao1=mean(S.chao1), sd_S.chao1=sd(S.chao1),
            mean_S.ACE=mean(S.ACE), sd_S.ACE=sd(S.ACE),
            mean_eshannon=mean(eshannon), sd_eshannon=sd(eshannon),
            mean_invsimpson=mean(invsimpson), sd_invsimpson=sd(invsimpson))
S.obs_season_fcam <- filter(estimators_indices_metadata, station=="FCaM") %>%
  group_by(season) %>%
  summarise(mean_S.obs=mean(S.obs), sd_S.obs=sd(S.obs),
            mean_S.chao1=mean(S.chao1), sd_S.chao1=sd(S.chao1),
            mean_S.ACE=mean(S.ACE), sd_S.ACE=sd(S.ACE),
            mean_eshannon=mean(eshannon), sd_eshannon=sd(eshannon),
            mean_invsimpson=mean(invsimpson), sd_invsimpson=sd(invsimpson))
S.obs_season_fca <- filter(estimators_indices_metadata, station=="FCa") %>%
  group_by(season) %>%
  summarise(mean_S.obs=mean(S.obs), sd_S.obs=sd(S.obs),
            mean_S.chao1=mean(S.chao1), sd_S.chao1=sd(S.chao1),
            mean_S.ACE=mean(S.ACE), sd_S.ACE=sd(S.ACE),
            mean_eshannon=mean(eshannon), sd_eshannon=sd(eshannon),
            mean_invsimpson=mean(invsimpson), sd_invsimpson=sd(invsimpson))

# Performing the Kruskal-Wallis H test
kruskal_S.obs <- kruskal.test(S.obs ~ station, data=estimators_indices_metadata)
kruskal_S.obs <- case_when(kruskal_S.obs$p.value < 0.05 & kruskal_S.obs$p.value >= 0.01 ~ 0.05,
                           kruskal_S.obs$p.value < 0.01 & kruskal_S.obs$p.value >= 0.001 ~ 0.01,
                           kruskal_S.obs$p.value < 0.001 & kruskal_S.obs$p.value >= 0.0001 ~ 0.001,
                           kruskal_S.obs$p.value < 0.0001 ~ 0.0001,
                           TRUE ~ kruskal_S.obs$p.value)
```

A total of `r samples_epiphytes` samples originating from epiphytic archaeal and bacterial communities associated with surfaces of the seagrass *C. nodosa* and the macroalga *C. cylindracea* were analysed. In addition, `r samples_seawater` samples (one of the samples was sequenced twice) originating from the ambient seawater were also processed for comparison. Generally, richness estimators and diversity indices showed similar trends. On average, higher values were found for *C. cylindracea* (mixed [Number of OTUs, `r S.obs_fcam$mean` ± `r S.obs_fcam$sd` OTUs] and monospecific [Number of OTUs, `r S.obs_fca$mean` ± `r S.obs_fca$sd` OTUs]) than for *C. nodosa* (Number of OTUs, `r S.obs_fcym$mean` ± `r S.obs_fcym$sd` OTUs) and lowest values were obtained for the microbial community of the ambient seawater (Number of OTUs, `r format(S.obs_f$mean, nsmall=1)` ± `r S.obs_f$sd` OTUs) (Kruskal-Wallis, *p* < `r kruskal_S.obs`) (\autoref{supp-calculators} and Tables \ref{supp-calculators_community_type} and \ref{supp-calculator_statistics}). Temporal changes did not reveal such large dissimilarities. *C. nodosa* communities showed a slow increase in all calculated richness estimators towards the end of the study, while *C. cylindracea* (mixed and monospecific) communities were characterized by slightly higher values in spring and summer than in autumn and winter (\autoref{supp-calculators}).

```{r seasonal_shared}
# Loading rarefied community data created during the generation of the richness and diversity calculators plot
load(file="results/numerical/rarefied.Rdata")

# Loading metadata 
metadata <- read_tsv("data/raw/metadata.csv")

# Joining metadata with OTU/sample data and summing sequences from each environment
rarefied_metadata <- inner_join(rarefied, metadata, by=c("Group"="ID")) %>%
  mutate(date=as.Date(date, "%d.%m.%Y")) %>%
  filter(date >= "2017-11-01") %>%
  select(!where(is.numeric) | where(~is.numeric(.x) && sum(.x)!=0)) %>%
  group_by(station) %>%
  summarise(across(starts_with("Otu"), sum), .groups="drop")

# Copying the sample labels to the rows (input for library vegan)
rarefied_metadata <- rarefied_metadata %>%
  column_to_rownames("station")

# Calculating matrices of similarity indices
jaccard <- as.matrix(vegdist(rarefied_metadata, method="jaccard", binary=T))
jaccard[lower.tri(jaccard, diag=TRUE)] <- NA
jaccard <- as_tibble(jaccard) %>%
  add_column("V1"=rownames(jaccard), .before=TRUE) %>%
  gather(key="V2", value="jaccard", 2:ncol(.)) %>%
  filter(!is.na(jaccard))

bray <- as.matrix(vegdist(rarefied_metadata, method="bray", binary=F))
bray[lower.tri(bray, diag=TRUE)] <- NA
bray <- as_tibble(bray) %>%
  add_column("V1"=rownames(bray), .before=TRUE) %>%
  gather(key="V2", value="bray", 2:ncol(.)) %>%
  filter(!is.na(bray))

similarity <- inner_join(jaccard, bray, by=c("V1"="V1", "V2"="V2")) %>%
  mutate(jaccard=1-jaccard) %>%
  mutate(bray=1-bray)
```

```{r seasonal_shared_temporal}
# Loading rarefied community data created during the generation of the richness and diversity calculators plot
load(file="results/numerical/rarefied.Rdata")

# Copying the sample labels to the rows (input for library vegan)
rarefied <- rarefied %>%
  column_to_rownames("Group")

# Calculating dissimilarity indices
jaccard <- as.matrix(vegdist(rarefied, method="jaccard", binary=T))
jaccard <- as_tibble(jaccard) %>%
  add_column("V1"=rownames(jaccard), .before=TRUE) %>%
  gather(key="V2", value="jaccard", 2:ncol(.))

bray <- as.matrix(vegdist(rarefied, method="bray", binary=F))
bray <- as_tibble(bray) %>%
  add_column("V1"=rownames(bray), .before=TRUE) %>%
  gather(key="V2", value="bray", 2:ncol(.))

distance <- inner_join(jaccard, bray, by=c("V1"="V1", "V2"="V2")) %>%
  mutate_at(c("V1", "V2"), list(~ as.character(.)))

# Loading metadata 
metadata <- read_tsv("data/raw/metadata.csv")

# Joining metadata with dissimilarity index data
distance_metadata <- inner_join(metadata, distance, by=c("ID"="V1")) %>%
  inner_join(., metadata, by=c("V2"="ID")) %>%
  mutate(date.x=as.Date(date.x, "%d.%m.%Y")) %>%
  mutate(date.y=as.Date(date.y, "%d.%m.%Y")) %>%
  mutate(jaccard=(1-jaccard)*100) %>%
  mutate(bray=(1-bray)*100) %>%
  gather(key="index", value="value", jaccard, bray)

# Filtering out consecutive sampling points
distance_metadata <- distance_metadata %>%
  filter((date.x=="2017-07-13" & date.y=="2017-07-27") |
         (date.x=="2017-07-27" & date.y=="2017-08-10") |
         (date.x=="2017-08-10" & date.y=="2017-08-24") |
         (date.x=="2017-08-24" & date.y=="2017-09-19") |
         (date.x=="2017-09-19" & date.y=="2017-10-12") |
         (date.x=="2017-10-12" & date.y=="2017-11-23") |
         (date.x=="2017-11-23" & date.y=="2017-12-04") |
         (date.x=="2017-12-04" & date.y=="2017-12-14") |
         (date.x=="2017-11-23" & date.y=="2017-12-14" & (station.x=="F" & station.y=="F")) |
         (date.x=="2017-12-14" & date.y=="2018-02-13") |
         (date.x=="2018-02-13" & date.y=="2018-03-27") |
         (date.x=="2017-12-14" & date.y=="2018-03-27" & (station.x=="FCaM" & station.y=="FCaM")) |
         (date.x=="2018-03-27" & date.y=="2018-04-24") |
         (date.x=="2018-04-24" & date.y=="2018-05-22") |
         (date.x=="2018-05-22" & date.y=="2018-06-19") |
         (date.x=="2018-06-19" & date.y=="2018-07-10") |
         (date.x=="2018-07-10" & date.y=="2018-08-09") |
         (date.x=="2018-08-09" & date.y=="2018-09-04") |
         (date.x=="2018-09-04" & date.y=="2018-10-05")) %>%
  filter((station.x=="F" & station.y=="F") |
         (station.x=="FCyM" & station.y=="FCyM") |
         (station.x=="FCaM" & station.y=="FCaM") |
         (station.x=="FCa" & station.y=="FCa"))

# Summarising the data
seasonal_shared_temporal <- distance_metadata %>%
  group_by(index, station.x) %>%
  summarise(mean=mean(value), sd=sd(value))

# Performing the Mann–Whitney U test
jaccard_test <- wilcox.test(filter(distance_metadata, index=="jaccard" & station.x=="F")$value,
                            filter(distance_metadata, index=="jaccard" & station.x!="F")$value)

jaccard_test <- case_when(jaccard_test$p.value < 0.05 & jaccard_test$p.value >= 0.01 ~ 0.05,
                          jaccard_test$p.value < 0.01 & jaccard_test$p.value >= 0.001 ~ 0.01,
                          jaccard_test$p.value < 0.001 & jaccard_test$p.value >= 0.0001 ~ 0.001,
                          jaccard_test$p.value < 0.0001 ~ 0.0001,
                          TRUE ~ jaccard_test$p.value)

bray_test <- wilcox.test(filter(distance_metadata, index=="bray" & station.x=="F")$value,
                            filter(distance_metadata, index=="bray" & station.x!="F")$value)

bray_test <- case_when(bray_test$p.value < 0.05 & bray_test$p.value >= 0.01 ~ 0.05,
                       bray_test$p.value < 0.01 & bray_test$p.value >= 0.001 ~ 0.01,
                       bray_test$p.value < 0.001 & bray_test$p.value >= 0.0001 ~ 0.001,
                       bray_test$p.value < 0.0001 ~ 0.0001,
                       TRUE ~ bray_test$p.value)
```

```{r core_otus}
# Loading rarefied community data created during the generation of the richness and diversity calculators plot
load(file="results/numerical/rarefied.Rdata")

# Joining metadata with OTU/sample data
metadata_rarefied <- read_tsv("data/raw/metadata.csv") %>%
  inner_join(., rarefied, by=c("ID"="Group")) %>%
  mutate(date=as.Date(date, "%d.%m.%Y"))

# Finding the core OTUs of each environment
# Cymodocea nodosa (mixed)
core <- metadata_rarefied %>%
  filter(station=="FCyM") %>%
  select_if(~ all(. != 0)) %>%
  select(starts_with("Otu"))
  
total <- metadata_rarefied %>%
  filter(station=="FCyM") %>%
  select(starts_with("Otu"))

core_otus_fcym <- length(core)/length(total) * 100
core_seq_fcym <- sum(core)/sum(total) * 100

# Caulerpa cylindracea (mixed)
core <- metadata_rarefied %>%
  filter(station=="FCaM") %>%
  select_if(~ all(. != 0)) %>%
  select(starts_with("Otu"))
  
total <- metadata_rarefied %>%
  filter(station=="FCaM") %>%
  select(starts_with("Otu"))

core_otus_fcam <- length(core)/length(total) * 100
core_seq_fcam <- sum(core)/sum(total) * 100

# Caulerpa cylindracea (monospecific)
core <- metadata_rarefied %>%
  filter(station=="FCa") %>%
  select_if(~ all(. != 0)) %>%
  select(starts_with("Otu"))
  
total <- metadata_rarefied %>%
  filter(station=="FCa") %>%
  select(starts_with("Otu"))

core_otus_fca <- length(core)/length(total) * 100
core_seq_fca <- sum(core)/sum(total) * 100

core <- tibble(core_otus=c(core_otus_fcym, core_otus_fcam, core_otus_fca),
               core_seq=c(core_seq_fcym, core_seq_fcam, core_seq_fca))
```

```{r anosim}
# Loading rarefied community data created during the generation of the richness and diversity calculators plot
load(file="results/numerical/rarefied.Rdata")

# Loading metadata 
metadata_rarefied <- read_tsv("data/raw/metadata.csv") %>%
  inner_join(., rarefied, by=c("ID"="Group"))

# Calculating ANOSIM
metadata_rarefied_select <- metadata_rarefied %>%
  select(starts_with("Otu"))
grouping <- metadata_rarefied %>%
  mutate(grouping=str_replace(station, "CaM$", "Ca"))
anosim_all <- anosim(metadata_rarefied_select, grouping$grouping, permutations=1000, distance="bray")
anosim_all_signif <- case_when(anosim_all$signif < 0.05 & anosim_all$signif >= 0.01 ~ 0.05,
                               anosim_all$signif < 0.01 & anosim_all$signif >= 0.001 ~ 0.01,
                               anosim_all$signif < 0.001 & anosim_all$signif >= 0.0001 ~ 0.001,
                               anosim_all$signif < 0.0001 ~ 0.0001,
                               TRUE ~ anosim_all$signif)

metadata_rarefied_select <- metadata_rarefied %>%
  filter(station=="FCyM") %>%
  select(starts_with("Otu"))
grouping <- metadata_rarefied %>%
  filter(station=="FCyM") %>%
  mutate(grouping=str_replace(season, "Autumn", "Winter"))
anosim_fcym <- anosim(metadata_rarefied_select, grouping$grouping, permutations=1000, distance="bray")
anosim_fcym_signif <- case_when(anosim_fcym$signif < 0.05 & anosim_fcym$signif >= 0.01 ~ 0.05,
                                anosim_fcym$signif < 0.01 & anosim_fcym$signif >= 0.001 ~ 0.01,
                                anosim_fcym$signif < 0.001 & anosim_fcym$signif >= 0.0001 ~ 0.001,
                                anosim_fcym$signif < 0.0001 ~ 0.0001,
                                TRUE ~ anosim_fcym$signif)

metadata_rarefied_select <- metadata_rarefied %>%
  filter(station=="FCaM" | station=="FCa") %>%
  select(starts_with("Otu"))
grouping <- metadata_rarefied %>%
  filter(station=="FCaM" | station=="FCa") %>%
  mutate(grouping=str_replace(season, "Autumn", "Winter")) %>%
  mutate(grouping=str_replace(grouping, "Spring", "Winter"))
anosim_fca <- anosim(metadata_rarefied_select, grouping$grouping, permutations=1000, distance="bray")
anosim_fca_signif <- case_when(anosim_fca$signif < 0.05 & anosim_fca$signif >= 0.01 ~ 0.05,
                                anosim_fca$signif < 0.01 & anosim_fca$signif >= 0.001 ~ 0.01,
                                anosim_fca$signif < 0.001 & anosim_fca$signif >= 0.0001 ~ 0.001,
                                anosim_fca$signif < 0.0001 ~ 0.0001,
                                TRUE ~ anosim_fca$signif)
```

A clear separation between ambient seawater and surface associated communities was found (\autoref{pcoa}). In addition, a separation of epiphytic bacterial and archaeal communities based on host species was detected. This separation was further supported by ANOSIM (R = `r format(round(anosim_all$statistic, 2), nsmall=2)`, *p* < `r anosim_all_signif`). The highest proportion of shared OTUs and community was found between mixed and monospecific  *C. cylindracea* (Jaccard, `r format(round(filter(similarity, V1=="FCa", V2=="FCaM")$jaccard, 2), nsmall=2)`; Bray-Curtis, `r format(round(filter(similarity, V1=="FCa", V2=="FCaM")$bray, 2), nsmall=2)`), while lower shared values were calculated between ambient seawater and epiphytic communities (Jaccard, `r format(round(min(filter(similarity, V1=="F")$jaccard), 2), nsmall=2)` -- `r format(round(max(filter(similarity, V1=="F")$jaccard), 2), nsmall=2)`; Bray-Curtis, `r format(round(min(filter(similarity, V1=="F")$bray), 2), nsmall=2)` -- `r format(round(max(filter(similarity, V1=="F")$bray), 2), nsmall=2)`). Shared proportions of OTUs and communities between *C. nodosa* and *C. cylindracea* (either mixed or monospecific) were approximately in-between the values obtained for the comparison of ambient seawater with all other communities and for the comparison of the mixed and monospecific *C. cylindracea* associated community. Seasonal changes of *C. nodosa* associated communities indicated a separation between spring, summer and autumn/winter samples (ANOSIM, R = `r format(round(anosim_fcym$statistic, 2), nsmall=2)`, *p* < `r anosim_fcym_signif`). For *C. cylindracea* associated communities a separation between summer and autumn/winter/spring samples was observed that was, however, not as strong as for *C. nodosa* associated communities (ANOSIM, R = `r format(round(anosim_fca$statistic, 2), nsmall=2)`, *p* < `r anosim_fca_signif`) (\autoref{pcoa}). Shared proportions of OTUs between consecutive sampling points were lower for ambient seawater (`r filter(seasonal_shared_temporal, index=="jaccard", station.x=="F")$mean` ± `r filter(seasonal_shared_temporal, index=="jaccard", station.x=="F")$sd` \si{\percent}) than for *C. nodosa* (`r filter(seasonal_shared_temporal, index=="jaccard", station.x=="FCyM")$mean` ± `r filter(seasonal_shared_temporal, index=="jaccard", station.x=="FCyM")$sd` \si{\percent}) and *C. cylindracea* (mixed [`r filter(seasonal_shared_temporal, index=="jaccard", station.x=="FCaM")$mean` ± `r filter(seasonal_shared_temporal, index=="jaccard", station.x=="FCaM")$sd` \si{\percent}] and monospecific [`r filter(seasonal_shared_temporal, index=="jaccard", station.x=="FCa")$mean` ± `r filter(seasonal_shared_temporal, index=="jaccard", station.x=="FCa")$sd` \si{\percent}]) associated communities (*p* < `r jaccard_test`), while mean proportions of shared communities between consecutive sampling points did not show such a difference (seawater, `r filter(seasonal_shared_temporal, index=="bray", station.x=="F")$mean` ± `r filter(seasonal_shared_temporal, index=="bray", station.x=="F")$sd` \si{\percent}; *C. nodosa*, `r filter(seasonal_shared_temporal, index=="bray", station.x=="FCyM")$mean` ± `r filter(seasonal_shared_temporal, index=="bray", station.x=="FCyM")$sd` \si{\percent}; *C. cylindracea* [mixed], `r filter(seasonal_shared_temporal, index=="bray", station.x=="FCaM")$mean` ± `r filter(seasonal_shared_temporal, index=="bray", station.x=="FCaM")$sd` \si{\percent}; *C. cylindracea* [monospecific], `r filter(seasonal_shared_temporal, index=="bray", station.x=="FCa")$mean` ± `r filter(seasonal_shared_temporal, index=="bray", station.x=="FCa")$sd` \si{\percent}) (*p* = `r bray_test`), although in ambient seawater higher fluctuations could be observed (\autoref{shared}). In addition, only `r min(core$core_otus)` -- `r max(core$core_otus)` \si{\percent} of OTUs from each surface associated community were present at all seasons. These persistent OTUs constituted a high proportion of total sequences (`r min(core$core_seq)` -- `r max(core$core_seq)` \si{\percent}) and were mainly contributing to abundant phylogenetic groups present throughout the year, e.g. the no relative *Rhodobacteraceae* in the case of *C. nodosa* or taxa within *Desulfobacterota* in the case of *C. cylindracea* (see below) (\autoref{supp-core_otus_taxonomy_table}).

```{r chloroplast}
# Loading input data containing sequence abundances and subsequent input data customization
community <- read_tsv("data/mothur/raw.trim.contigs.good.unique.good.filter.unique.precluster.pick.nr_v138.wang.tax.summary") %>%
  filter(!str_detect(taxon, "^Eukaryota")) %>%
  filter(taxon!="Root")

# Remove mitochondrial sequences and subtract their number from higher taxonomic levels to which
# they belong
mitochondria <- filter(community, str_detect(taxon, "^Mitochondria$"))$rankID
community <- mutate_at(community, 5:ncol(community), list(~case_when(
    rankID==str_extract(mitochondria, "(\\d+\\.){4}\\d+") ~ . - .[taxon=="Mitochondria"],
    rankID==str_extract(mitochondria, "(\\d+\\.){3}\\d+") ~ . - .[taxon=="Mitochondria"],
    rankID==str_extract(mitochondria, "(\\d+\\.){2}\\d+") ~ . - .[taxon=="Mitochondria"],
    rankID==str_extract(mitochondria, "(\\d+\\.){1}\\d+") ~ . - .[taxon=="Mitochondria"],
    TRUE ~ .))) %>%
  filter(!str_detect(taxon, "^Mitochondria")) %>%
  select(-ATCC_1, -ATCC_5, -NC_1) %>%
  mutate(`23`=`23_1`+`23_2`) %>%
  select(-`23_1`, -`23_2`) %>%
  group_by(taxlevel) %>%
  mutate_at(5:ncol(.), list(~. / sum(.) * 100)) %>%
  ungroup()

# Selection of groups for plotting
plot <- community %>%
  filter(taxon=="Chloroplast") %>%
  bind_rows(summarise_all(., list(~ifelse(is.numeric(.), 100-sum(.), paste("Other")))))

# Loading colors for each group on the plot
color <- read_tsv("data/raw/group_colors.csv") %>%
  select(-Taxlevel) %>%
  deframe()

# Generation of italic names for groups
names <- parse(text=case_when(plot$taxon=="Chloroplast" ~ paste0("plain('", plot$taxon,  "')"),
                              plot$taxon=="Bacteria_unclassified" ~ "italic('Bacteria')~plain('(No Relative)')",
                              plot$taxon=="Marinimicrobia_(SAR406_clade)" ~ "italic('Marinimicrobia')",
                              plot$taxon=="Other" ~ paste0("plain('", plot$taxon, "')"),
                              plot$taxon=="No Relative" ~ paste0("plain('", plot$taxon, "')"),
                              TRUE ~ paste0("italic('", plot$taxon, "')")))

# Tidying the sequence abundance data
plot <- gather(plot, key="Group", value="abundance", 6:(ncol(plot)))

# Loading metadata
metadata <- read_tsv("data/raw/metadata.csv") %>%
  filter(ID!="23_1") %>%
  mutate(ID=str_replace(ID, "23_2", "23")) %>%
  mutate(label=str_replace(label, "24/4-18 F-2", "24/4-18 F"))

# Joining sequence abundances data and metadata
plot <- inner_join(metadata, plot, by=c("ID"="Group")) %>%
  mutate(taxon=factor(taxon, levels=rev(unique(plot$taxon)))) %>%
  mutate(label=factor(label, levels=metadata$label)) %>%
  mutate(date=as.Date(date, "%d.%m.%Y"))

# Autumn/Winter and Spring/Summer abundances
chloroplast <- plot %>%
  filter(station!="F" & taxon!="Other") %>%
  mutate(season=str_replace(season, "Autumn", "Winter")) %>%
  mutate(season=str_replace(season, "Spring", "Summer")) %>%
  group_by(season) %>%
  summarise(mean=mean(abundance), sd=sd(abundance))

# Performing the Mann–Whitney U test
chloroplast_test <- plot %>%
  filter(station!="F" & taxon!="Other") %>%
  mutate(season=str_replace(season, "Autumn", "Winter")) %>%
  mutate(season=str_replace(season, "Spring", "Summer")) %>%
  select(taxon, season, abundance)

chloroplast_test <- wilcox.test(filter(chloroplast_test, season=="Winter")$abundance,
                                filter(chloroplast_test, season=="Summer")$abundance)

chloroplast_test <- case_when(chloroplast_test$p.value < 0.05 & chloroplast_test$p.value >= 0.01 ~ 0.05,
                              chloroplast_test$p.value < 0.01 & chloroplast_test$p.value >= 0.001 ~ 0.01,
                              chloroplast_test$p.value < 0.001 & chloroplast_test$p.value >= 0.0001 ~ 0.001,
                              chloroplast_test$p.value < 0.0001 ~ 0.0001,
                              TRUE ~ chloroplast_test$p.value)
```

The taxonomic composition of both, macrophyte associated and ambient seawater community, was dominated by bacterial (`r filter(archaea_bacteria, taxon=="Bacteria")$mean` ± `r filter(archaea_bacteria, taxon=="Bacteria")$sd` \si{\percent}) over archaeal sequences (`r filter(archaea_bacteria, taxon=="Archaea")$mean` ± `r filter(archaea_bacteria, taxon=="Archaea")$sd` \si{\percent}) (\autoref{community}). Higher relative abundances of chloroplast related sequences were only observed in surface associated communities, with higher values in autumn/winter (`r filter(chloroplast, season=="Winter")$mean` ± `r filter(chloroplast, season=="Winter")$sd` \si{\percent}) than in spring/summer (`r filter(chloroplast, season=="Summer")$mean` ± `r filter(chloroplast, season=="Summer")$sd` \si{\percent}) (*p* < `r chloroplast_test`) (\autoref{supp-chloroplast}). Generally, at higher taxonomic ranks (phylum-class), epiphytic and ambient seawater microbial communities were composed of similar bacterial taxa. Ambient seawater communities were mainly comprised of *Actinobacteriota*, *Bacteroidota*, *Cyanobacteria*, *Alphaproteobacteria*, *Gammaproteobacteria* and *Verrucomicrobiota*. Communities associated with *C. nodosa* consisted additionally of *Planctomycetota* contributing more in summer 2018 than in other seasons. In addition, communities from mixed and monospecific *C. cylindracea* were similar and characterized by the same groups as ambient seawater and *C. nodosa* communities with the addition of *Desulfobacterota* (\autoref{community}). Larger differences between environments and host species were observed at lower taxonomic ranks (Figures \ref{cyano} -- \ref{desulfo}).

```{r cyano}
# Loading input data containing sequence abundances and subsequent input data customization
community <- read_tsv("data/mothur/raw.trim.contigs.good.unique.good.filter.unique.precluster.pick.nr_v138.wang.tax.summary") %>%
  filter(!str_detect(taxon, "^Eukaryota")) %>%
  filter(taxon!="Root")

# Remove chloroplast and mitochondrial sequences and subtract their number from higher taxonomic levels to which
# they belong
chloroplast <- filter(community, str_detect(taxon, "^Chloroplast$"))$rankID
mitochondria <- filter(community, str_detect(taxon, "^Mitochondria$"))$rankID
community <- mutate_at(community, 5:ncol(community), list(~case_when(
  rankID==str_extract(chloroplast, "(\\d+\\.){3}\\d+") ~ . - .[taxon=="Chloroplast"],
  rankID==str_extract(chloroplast, "(\\d+\\.){2}\\d+") ~ . - .[taxon=="Chloroplast"],
  rankID==str_extract(chloroplast, "(\\d+\\.){1}\\d+") ~ . - .[taxon=="Chloroplast"],
  TRUE ~ .))) %>%
  filter(!str_detect(taxon, "^Chloroplast")) %>%
  mutate_at(5:ncol(.), list(~case_when(
    rankID==str_extract(mitochondria, "(\\d+\\.){4}\\d+") ~ . - .[taxon=="Mitochondria"],
    rankID==str_extract(mitochondria, "(\\d+\\.){3}\\d+") ~ . - .[taxon=="Mitochondria"],
    rankID==str_extract(mitochondria, "(\\d+\\.){2}\\d+") ~ . - .[taxon=="Mitochondria"],
    rankID==str_extract(mitochondria, "(\\d+\\.){1}\\d+") ~ . - .[taxon=="Mitochondria"],
    TRUE ~ .))) %>%
  filter(!str_detect(taxon, "^Mitochondria")) %>%
  select(-ATCC_1, -ATCC_5, -NC_1) %>%
  mutate(`23`=`23_1`+`23_2`) %>%
  select(-`23_1`, -`23_2`) %>%
  group_by(taxlevel) %>%
  mutate_at(5:ncol(.), list(~. / sum(.) * 100)) %>%
  ungroup()

# Selection of groups for plotting
select <- filter(community,
               taxlevel==6 &
               str_detect(rankID, paste0(filter(community, str_detect(taxon, "^Cyanobacteria$"))$rankID, "\\."))) %>%
  filter_at(6:ncol(.), any_vars(. >= 1))

plot <- filter(community,
               taxlevel==6 &
               str_detect(rankID, paste0(filter(community, str_detect(taxon, "^Cyanobacteria$"))$rankID, "\\."))) %>%
  mutate_at(5:ncol(.), list(~. / sum(.) * 100)) %>%
  ungroup() %>%
  filter(taxon %in% select$taxon) %>%
  bind_rows(summarise_all(., list(~ifelse(is.numeric(.), 100-sum(.), paste("Other_Cyanobacteria")))))

# Loading colors for each group on the plot
color <- read_tsv("data/raw/group_colors.csv") %>%
  select(-Taxlevel) %>%
  deframe()

# Generation of italic names for taxonomic groups
names <- parse(text=case_when(plot$taxon=="Cyanobacteriales_unclassified" ~ "italic('Cyanobacteriales')~plain('(NR)')",
                              plot$taxon=="Spirulina_DRTO-55.2" ~ "italic('Spirulina')",
                              plot$taxon=="Pleurocapsa_PCC-7319" ~ "italic('Pleurocapsa')",
                              plot$taxon=="Cyanobacteriia_unclassified" ~
                                "italic('Cyanobacteriia')~plain('(NR)')",
                              plot$taxon=="Nodosilineaceae_unclassified" ~ "italic('Nodosilineaceae')~plain('(NR)')",
                              plot$taxon=="Acrophormium_PCC-7375" ~ "italic('Acrophormium')",
                              plot$taxon=="Phormidesmis_ANT.LACV5.1" ~ "italic('Phormidesmis')",
                              plot$taxon=="Phormidium_MBIC10003" ~ "italic('Phormidium')",
                              plot$taxon=="Cyanobium_PCC-6307" ~ "italic('Cyanobium')",
                              plot$taxon=="Synechococcus_CC9902" ~ "italic('Synechococcus')",
                              plot$taxon=="Schizothrix_LEGE_07164" ~ "italic('Schizothrix')",
                              plot$taxon=="Thermosynechococcales_unclassified" ~ "italic('Thermosynechococcales')~plain('(NR)')",  
                              plot$taxon=="Other_Cyanobacteria" ~ "plain('Other')~italic('Cyanobacteria')",
                              TRUE ~ paste0("italic('", plot$taxon, "')")))

# Tidying the sequence abundance data
plot <- gather(plot, key="Group", value="abundance", 6:ncol(plot))

# Loading metadata
metadata <- read_tsv("data/raw/metadata.csv") %>%
  filter(ID!="23_1") %>%
  mutate(ID=str_replace(ID, "23_2", "23")) %>%
  mutate(label=str_replace(label, "24/4-18 F-2", "24/4-18 F"))

# Joining sequence abundances data and metadata
plot <- inner_join(metadata, plot, by=c("ID"="Group")) %>%
  mutate(taxon=factor(taxon, levels=unique(plot$taxon))) %>%
  mutate(label=factor(label, levels=metadata$label)) %>%
  mutate(date=as.Date(date, "%d.%m.%Y"))

# Selecting the relative abundance of the targeted group in the whole community and tidying the obtained data
whole <- filter(community, taxlevel==2) %>%
  gather(key="Group", value="abundance", 6:ncol(.)) %>%
  filter(taxon=="Cyanobacteria")

# Calculating the mean and standard deviation of all samples
cyano_all_whole <- inner_join(metadata, whole, by=c("ID"="Group")) %>%
  mutate(date=as.Date(date, "%d.%m.%Y")) %>%
  summarise(mean=mean(abundance), sd=sd(abundance))

# Calculating the mean and standard deviation for each season and station
cyano_station_season_whole <- inner_join(metadata, whole, by=c("ID"="Group")) %>%
  mutate(date=as.Date(date, "%d.%m.%Y")) %>%
  group_by(station, season) %>%
  summarise(mean=mean(abundance), sd=sd(abundance))

# Performing the Mann–Whitney U test
cyano_station_season_whole_test <- inner_join(metadata, whole, by=c("ID"="Group")) %>%
  mutate(date=as.Date(date, "%d.%m.%Y"))

cyano_station_season_whole_test <- wilcox.test(filter(cyano_station_season_whole_test, season=="Autumn" & station!="F")$abundance,
                                               filter(cyano_station_season_whole_test, season!="Autumn" & station!="F")$abundance)

cyano_station_season_whole_test <- case_when(cyano_station_season_whole_test$p.value < 0.05 & cyano_station_season_whole_test$p.value >= 0.01 ~ 0.05,
                                             cyano_station_season_whole_test$p.value < 0.01 & cyano_station_season_whole_test$p.value >= 0.001 ~ 0.01,
                                             cyano_station_season_whole_test$p.value < 0.001 & cyano_station_season_whole_test$p.value >= 0.0001 ~ 0.001,
                                             cyano_station_season_whole_test$p.value < 0.0001 ~ 0.0001,
                                             TRUE ~ cyano_station_season_whole_test$p.value)
```

*Cyanobacteria* related sequences comprised, on average, `r cyano_all_whole$mean` ± `r cyano_all_whole$sd` \si{\percent} of total sequences (\autoref{cyano}). Higher proportions were found for *C. nodosa* (`r filter(cyano_station_season_whole, station=="FCyM" & season=="Autumn")$mean` ± `r filter(cyano_station_season_whole, station=="FCyM" & season=="Autumn")$sd` \si{\percent}) and *C. cylindracea* mixed (`r filter(cyano_station_season_whole, station=="FCaM" & season=="Autumn")$mean` ± `r filter(cyano_station_season_whole, station=="FCaM" & season=="Autumn")$sd` \si{\percent}) and monospecific (`r filter(cyano_station_season_whole, station=="FCa" & season=="Autumn")$mean` ± `r filter(cyano_station_season_whole, station=="FCa" & season=="Autumn")$sd` \si{\percent}) associated communities in autumn (*p* < `r cyano_station_season_whole_test`) and for ambient seawater communities in winter (`r filter(cyano_station_season_whole, station=="F" & season=="Winter")$mean` ± `r filter(cyano_station_season_whole, station=="F" & season=="Winter")$sd` \si{\percent}). Large taxonomic differences between surface associated and ambient seawater cyanobacterial communities were observed. Ambient seawater communities were mainly comprised of *Cyanobium* and *Synechococcus*, while surface associated communities were comprised of *Pleurocapsa* and sequences within the class *Cyanobacteriia* that could not be further classified (no relative *Cyanobacteriia*) (\autoref{cyano}). In addition, seasonal changes in surface associated cyanobacterial communities were observed with *Pleurocapsa* and no relative *Cyanobacteriia* comprising larger proportions of *Cyanobacteria* in autumn and winter and *Acrophormium*, *Phormidesmis* and sequences without known relatives within the *Nodosilineaceae* (no relative *Nodosilineaceae*) in spring and summer (\autoref{cyano}).

```{r bactero}
# Selecting the relative abundance of the targeted group in the whole community and tidying the obtained data
whole <- filter(community, taxlevel==2) %>%
  gather(key="Group", value="abundance", 6:ncol(.)) %>%
  filter(taxon=="Bacteroidota")

# Calculating the mean and standard deviation of all samples
bactero_all_whole <- inner_join(metadata, whole, by=c("ID"="Group")) %>%
  mutate(date=as.Date(date, "%d.%m.%Y")) %>%
  summarise(mean=mean(abundance), sd=sd(abundance))
```

Sequences classified as *Bacteroidota* comprised, on average, `r bactero_all_whole$mean` ± `r bactero_all_whole$sd` \si{\percent} of all sequences (\autoref{bactero}). Similar to *Cyanobacteria*, large differences in the taxonomic composition between ambient seawater and surface associated communities were found (\autoref{bactero}). The ambient seawater community was characterized by the NS4 and NS5 marine groups, uncultured *Cryomorphaceae*, uncultured *Flavobacteriaceae*, NS11-12 marine group, *Balneola*, uncultured *Balneolaceae* and *Formosa*. In contrast, in surface associated communities *Lewinella*, *Portibacter*, *Rubidimonas*, sequences without known relatives within the *Saprospiraceae* (no relative *Saprospiraceae*), uncultured *Saprospiraceae*, sequences without known relatives within the *Flavobacteriaceae* (no relative *Flavobacteriaceae*) and uncultured *Rhodothermaceae* were found. Some groups showed minor seasonal changes such as no relative *Flavobacteriaceae* whose sequences were more abundant from November 2017 until June 2018. In contrast, uncultured *Rhodothermaceae* showed higher proportions from June 2018 until the end of the study period. Surface associated *Bacteroidota* communities were very diverse as observed in the high proportion of taxa clustering as other *Bacteroidota* (\autoref{bactero}).

```{r alpha}
# Selection of groups for plotting
select <- filter(community,
               taxlevel==6 &
               str_detect(rankID, paste0(filter(community, str_detect(taxon, "^Alphaproteobacteria$"))$rankID, "\\."))) %>%
  filter_at(6:ncol(.), any_vars(. >= 2))

plot <- filter(community,
               taxlevel==6 &
               str_detect(rankID, paste0(filter(community, str_detect(taxon, "^Alphaproteobacteria$"))$rankID, "\\."))) %>%
  mutate_at(5:ncol(.), list(~. / sum(.) * 100)) %>%
  ungroup() %>%
  filter(rankID %in% select$rankID) %>%
  bind_rows(summarise_all(., list(~ifelse(is.numeric(.), 100-sum(.), paste("Other_Alphaproteobacteria"))))) %>%
# Remove last digit from rankID so for the next step
  mutate(rankID=if_else(taxon=="uncultured", str_replace(rankID, "\\.\\d+$", ""), rankID))

# Adding data to "uncultured" taxa describing the higher taxonomic level to which they belong
uncultured <- select(filter(community, rankID %in% filter(plot, taxon=="uncultured")$rankID), rankID, taxon) %>%
  rename(rankID_uncultured=rankID, taxon_uncultured=taxon)

plot <- left_join(plot, uncultured, by=c("rankID"="rankID_uncultured")) %>%
  mutate(taxon=if_else(taxon=="uncultured", paste0(taxon, "_", taxon_uncultured), taxon)) %>%
  select(-taxon_uncultured)

# Loading colors for each group on the plot
color <- read_tsv("data/raw/group_colors.csv") %>%
  select(-Taxlevel) %>%
  deframe()

# Generation of italic names for taxonomic groups
names <- parse(text=case_when(str_detect(plot$taxon, "uncultured") ~ paste0("plain('Uncultured')~italic('", str_remove(plot$taxon, "uncultured_"), "')"),
                              str_detect(plot$taxon, "unclassified") ~ paste0("italic('", str_remove(plot$taxon, "_unclassified"), "')~plain('(NR)')"),
                              plot$taxon=="OCS116_clade_ge" ~ "plain('OCS116 Clade')",
                              plot$taxon=="Candidatus_Puniceispirillum" ~ "italic('\"Candidatus')~plain('Puniceispirillum\"')",
                              plot$taxon=="SAR116_clade_ge" ~ "plain('SAR116 Clade')",
                              plot$taxon=="Stappiaceae_ge" ~ "italic('Stappiaceae')",
                              plot$taxon=="HIMB11" ~ "plain('HIMB11')",
                              plot$taxon=="AEGEAN-169_marine_group_ge" ~ "plain('AEGEAN-169 Marine Group')",
                              plot$taxon=="Clade_Ia" ~ "plain('Clade Ia')",
                              plot$taxon=="Clade_Ib" ~ "plain('Clade Ib')",
                              plot$taxon=="Clade_II_ge" ~ "plain('Clade II')",
                              plot$taxon=="Clade_III_ge" ~ "plain('Clade III')",
                              plot$taxon=="Other_Alphaproteobacteria" ~ "plain('Other')~italic('Alphaproteobacteria')",
                              TRUE ~ paste0("italic('", plot$taxon, "')")))

# Tidying the sequence abundance data
plot <- gather(plot, key="Group", value="abundance", 6:ncol(plot))

# Loading metadata
metadata <- read_tsv("data/raw/metadata.csv") %>%
  filter(ID!="23_1") %>%
  mutate(ID=str_replace(ID, "23_2", "23")) %>%
  mutate(label=str_replace(label, "24/4-18 F-2", "24/4-18 F"))

# Joining sequence abundances data and metadata
plot <- inner_join(metadata, plot, by=c("ID"="Group")) %>%
  mutate(taxon=factor(taxon, levels=unique(plot$taxon))) %>%
  mutate(label=factor(label, levels=metadata$label)) %>%
  mutate(date=as.Date(date, "%d.%m.%Y"))

# Selecting the relative abundance of the targeted group in the whole community and tidying the obtained data
whole <- filter(community, taxlevel==3) %>%
  gather(key="Group", value="abundance", 6:ncol(.)) %>%
  filter(taxon=="Alphaproteobacteria")

# Calculating the mean and standard deviation of all samples
alpha_all_whole <- inner_join(metadata, whole, by=c("ID"="Group")) %>%
  mutate(date=as.Date(date, "%d.%m.%Y")) %>%
  summarise(mean=mean(abundance), sd=sd(abundance))

# Calculating the mean abundance and standard deviation of Rhodobacteraceae_unclassified within Alphaproteobacteria
rhodo <- plot %>%
  filter(station!="F" & taxon=="Rhodobacteraceae_unclassified") %>%
  summarise(mean=mean(abundance), sd=sd(abundance))
```

On average, *Alphaproteobacteria* were in comparison to the other high rank taxa the largest taxonomic group, comprising `r alpha_all_whole$mean` ± `r alpha_all_whole$sd` \si{\percent} of all sequences (\autoref{alpha}). In accordance to the above described taxa, large differences between ambient seawater and surface associated communities were observed. Ambient seawater communities were composed mainly of the SAR11 clade, AEGEAN-169 marine group, SAR116 clade, sequences without known relatives within the *Rhodobacteraceae* (no relative *Rhodobacteraceae*), HIMB11 and the OCS116 clade, while surface associated communities were composed mainly of no relative *Rhodobacteraceae* and to a lesser degree of *Pseudoahrensia*, *Amylibacter* and sequences without known relatives within the *Alphaproteobacteria* (no relative *Alphaproteobacteria*) and *Hyphomonadaceae* (no relative  *Hyphomonadaceae*). Representatives of no relative *Rhodobacteraceae* comprised on average `r rhodo$mean` ± `r rhodo$sd` \si{\percent} of all alphaproteobacterial sequences in the epiphytic community (\autoref{alpha}). In addition, *Amylibacter* was detected mainly in *C. nodosa* from November 2017 until March 2018.

```{r gamma}
# Selecting the relative abundance of the targeted group in the whole community and tidying the obtained data
whole <- filter(community, taxlevel==3) %>%
  gather(key="Group", value="abundance", 6:ncol(.)) %>%
  filter(taxon=="Gammaproteobacteria")

# Calculating the mean and standard deviation of all samples
gamma_all_whole <- inner_join(metadata, whole, by=c("ID"="Group")) %>%
  mutate(date=as.Date(date, "%d.%m.%Y")) %>%
  summarise(mean=mean(abundance), sd=sd(abundance))
```

Sequences related to *Gammaproteobacteria* comprised on average `r gamma_all_whole$mean` ± `r gamma_all_whole$sd` \si{\percent} of all sequences (\autoref{gamma}). Similar to above mentioned taxa, large taxonomic differences between ambient seawater and surface associated communities were found. Ambient seawater communities were mainly comprised of the OM60 (NOR5) clade, *Litoricola*, *Acinetobacter* and the SAR86 clade, while epiphytic communities were mainly composed of sequences without known relatives within the *Gammaproteobacteria* (no relative *Gammaproteobacteria*) and *Granulosicoccus*. Beside these two groups specific to all three epiphytic communities, *C. nodosa* was characterized by *Arenicella*, *Methylotenera* and sequences without known relatives within the *Burkholderiales* (no relative *Burkholderiales*), while *Thioploca*, *Reinekea* and sequences without known relatives within *Cellvibrionaceae* (no relative *Cellvibrionaceae*) were more specific to both mixed and monospecific *C. cylindracea*. In addition, *Arenicella* was more pronounced in November and December 2017, while no relative *Burkholderiales* and *Methylotenera* were characteristic for the period from March until May 2018. For the *C. cylindracea* specific taxa no relative *Cellvibrionaceae* and *Reinekea* showed seasonality and were characteristic for samples originating from June to October 2018. In addition, similar to *Bacteroidota*, a large proportion of the surface associated community was grouped as other *Gammaproteobacteria* indicating high diversity within this group (\autoref{gamma}).

```{r delta}
# Selecting the relative abundance of the targeted group in the whole community and tidying the obtained data
whole <- filter(community, taxlevel==2) %>%
  gather(key="Group", value="abundance", 6:ncol(.)) %>%
  filter(taxon=="Desulfobacterota")

# Calculating the mean and standard deviation for each station
desulfo_station_whole <- inner_join(metadata, whole, by=c("ID"="Group")) %>%
  mutate(date=as.Date(date, "%d.%m.%Y")) %>%
  group_by(station) %>%
  summarise(mean=mean(abundance), sd=sd(abundance))
```

*Desulfobacterota* were specific for *C. cylindracea*. In the mixed and monospecific *C. cylindracea* communities the proportion of *Desulfobacterota* was `r filter(desulfo_station_whole, station=="FCaM")$mean` ± `r filter(desulfo_station_whole, station=="FCaM")$sd` \si{\percent} and `r filter(desulfo_station_whole, station=="FCa")$mean` ± `r filter(desulfo_station_whole, station=="FCa")$sd` \si{\percent}, respectively (\autoref{desulfo}). In contrast, in ambient seawater and *C. nodosa* communities the contribution of *Desulfobacterota* was only `r filter(desulfo_station_whole, station=="F")$mean` ± `r filter(desulfo_station_whole, station=="F")$sd` \si{\percent} and `r filter(desulfo_station_whole, station=="FCyM")$mean` ± `r filter(desulfo_station_whole, station=="FCyM")$sd` \si{\percent}, respectively. In *C. cylindracea* the community consisted mainly of *Desulfatitalea*, *Desulfobulbus*, *Desulfopila*, *Desulforhopalus*, *Desulfotalea*, SEEP-SRB4, uncultured *Desulfocapsaceae* and sequences without known relatives within the *Desulfobacteraceae* (no relative *Desulfobacteraceae*), *Desulfobulbaceae* (no relative *Desulfobulbaceae*) and *Desulfocapsaceae* (no relative *Desulfocapsaceae*) (\autoref{desulfo}).

\newpage
## Discussion
In the present study, we applied a selective epiphytic DNA isolation procedure based on direct cellular lysis [@Korlevic2021] coupled with a monthly sampling and Illumina amplicon sequencing to describe in detail the bacterial and archaeal communities associated with the surfaces of two marine macrophytes, *C. nodosa* and *C. cylindracea*. Highest richness was observed for *C. cylindracea* (mixed and monospecific) followed by *C. nodosa* and lowest richness was found in ambient seawater microbial communities. Higher richness of microbial communities associated with macrophytes than in ambient seawater has been described earlier [@Ugarelli2019; @Mancuso2016] and could be attributed to a larger set of inhabitable microniches existing on macrophyte surfaces than in the ambient seawater. The highest richness observed for *C. cylindracea* might be partly due to its contact with the sediment. The stolon of *C. cylindracea* is attached to the sediment surface with rhizoids and thus, the stolon and rhizoids are in direct contact with the sediment. Also, studies have shown that the presence of *C. cylindracea* can alter the content and biochemical composition of sedimentary organic matter [@Rizzo2017; @Pusceddu2016; @Rizzo2020] possibly further expanding the number of inhabitable microniches and thus causing the observed increase in richness. Seasonal differences in richness observed for surface attached communities indicated a slightly higher richness in spring and summer. This pattern could be explained by a more intense macrophyte growth in these two seasons than in autumn and winter [@Najdek2020a; @Zavodnik1998; @Ruitton2005]. During their main growth season in spring and summer macrophytes exhibit a more dynamic chemical interaction with the surface community probably causing an increase in the number of inhabitable microniches [@Rickert2016; @Borges2015]. Proportions of shared epiphytic OTUs between consecutive sampling points were low also indicated by the proportion of OTUs ($\leq$ `r max(core$core_otus)` \si{\percent}) present at every sampling date (\autoref{shared}). These persistent OTUs, however, accounted for a high proportion of sequences ($\geq$ `r min(core$core_seq)` \si{\percent}), as is often the case with similar high-frequency sampling studies [@Gilbert2009; @Gilbert2012]. In comparison to the seawater community, higher values of shared OTUs between consecutive sampling points were observed for the macrophyte surface associated communities. It appears that macrophyte surfaces are providing more stable conditions than the ambient seawater.

We observed a strong differentiation between the surface attached and ambient seawater communities at the level of OTUs which is in agreement with most published studies [@Ugarelli2019; @Roth-Schulze2016; @Burke2011a; @Michelou2013; @Crump2018; @Mancuso2016; @Sanders-Smith2020]. This indicates that marine macrophytes are selecting microorganisms from the pool of microbial taxa present in the ambient seawater, modifying the microbial community once the macrophyte associated microbial biofilm develops [@Salaun2012; @Michelou2013]. In addition, similar to the study of @Roth-Schulze2016 seagrass and macroalgae specific microbial communities were identified, while no difference between *C. cylindracea* settlements was observed indicating that seagrass and macroalgae specific metabolism is involved in the selection and development of the associated biofilm. At the level of OTUs seasonal changes of *C. nodosa* and *C. cylindracea* associated communities were identified that could be linked to the growth cycle of the seagrass and macroalgae [@Najdek2020a; @Agostini2003]. *C. nodosa* was characterized by a spring community during maximum seagrass proliferation, a summer community during the highest standing stock of *C. nodosa* and an autumn/winter community during the decay of seagrass biomass. In contrast, *C. cylindracea* started to proliferate in late spring and was characterized only by a summer community during high growth rates and by an autumn/winter/spring community when the biomass was at the peak and decaying thereafter. Similar seasonal changes in the epiphytic community have also been described for other macroalgae [@Lachnit2011; @Tujula2010].

The taxonomic analysis showed higher chloroplast sequence abundances in autumn/winter than in spring/summer. This pattern is not surprising as seagrasses harbour more algal epiphytes during autumn/winter than in spring/summer [@Reyes2001]. Furthermore, we used an adapted DNA isolation protocol that is known to partially co-extract DNA from planktonic eukaryotes [@Korlevic2015]. In general, the taxonomic analysis identified epiphytic phylogenetic groups present throughout the year comprising most of the reads, and taxa present in lower proportions showing seasonal patterns. The first group was comprised of members of the *Bacteroidota* family *Saprospiraceae*, the alphaproteobacterial *Rhodobacteraceae* and *Hyphomonadaceae*, the gammaproteobacterial genus *Granulosicoccus*, sequences without known relatives within *Gammaproteobacteria* and various taxa within *Desulfobacterota* (Figures \ref{bactero} -- \ref{desulfo}). All these groups were found on all host species, with the exception of *Desulfobacterota* that was characteristic for *C. cylindracea*. In addition, the persistence of *Rhodobacteraceae* in the case of *C. nodosa* and *Desulfobacterota* in the case of *C. cylindracea* could be observed in the taxonomic classification of OTUs present at every sampling date. Within the *Bacteroidota* different groups within *Saprospiraceae* (e.g. *Lewinella*, *Portibacter* and *Rubidimonas*) were identified to be persistent. It has been suggested that members of this family are important in the hydrolysis and utilization of complex organic sources [@McIlroy2014]. Surface attached life style would be beneficial to these microbes as they could thrive on products of host cellular breakdown or by-products of host metabolism, so it not surprising that they are often found associated with macrophyte surfaces [@Burke2011a; @Crump2018; @McIlroy2014]. *Rhodobacteraceae* are often detected on macrophyte surfaces and usually are one of the most abundant groups [@Burke2011a; @Michelou2013; @Mancuso2016]. The functional association between macrophytes and members of this groups is difficult to assess based on 16S rRNA analysis as this family is phenotypically, metabolically, and ecologically very diverse [@Pujalte2014]. However, some interesting metabolic capacities linked to this group were described. Genomic analysis of *Rhodobacteraceae* strains and metatranscriptomic sequencing of seagrass microbiomes revealed the potential for biosynthesis of indole-3-acetic acid (IAA), a plant hormone [@Simon2017], indicating a possible intake by seagrasses. However, another study found no effect of IAA on *C. nodosa* growth showing the complexity of macrophyte--microbes interactions [@Munoz1995]. Another persistent alphaproteobacterial family was the *Hyphomonadaceae*, a group that contain species with stalks used to attach cells to different surfaces [@Abraham2014]. This group has been previously associated with seagrass surfaces [@Weidner2000] and it is believed that possessing stalks could be an advantage to keep the cells in the proximity of exudate excreted by the host [@Abraham2014; @Weidner2000].

Within the *Gammaproteobacteria*, sequences without known representatives were the most pronounced group present throughout the year. *Gammaproteobacteria* are often a major constituent of macrophyte epiphytic communities [@Michelou2013; @Burke2011a; @Crump2018]. A study has attributed the expression of enzymes for the degradation of galactose-based algal polymers to this class indicating their possible involvement into epibiotic algal biofilm control [@Crump2018]. In addition, *Granulosicoccus* was also found in almost all samples. A species of this genus has been isolated from the leaf surface of the seagrass *Zostera marina* [@Kurilenko2010], while sequences related to this genus have been found on the surfaces of macroalgae [@Bengtsson2012; @Lachnit2011], including *C. cylindracea* [@Rizzo2016], indicating this group preference for macrophyte surfaces. It is possible that bacteria of this genus can thrive on exudates of different macrophytes as it is known from cultivated members that they can utilise various sugars and amino acids [@Ivanova2014]. The presence of *Desulfobacterota* only on *C. cylindracea* is to be expected as part of the epiphytic community is in direct contact with the sediment. The *Desulfobacterota* community was comprised of known sulphate sediment groups such as the *Desulfatitalea* and no relative *Desulfocapsaceae* [@Higashioka2015; @Kuever2014]. Sequences related to sulphur cycling bacterial groups have been found in *Caulerpa* endophytic and epiphytic communities [@Aires2013]. It is possible that these groups are involved into enhanced sulphate reduction rates observed in sediments underlying *Caulerpa* settlements causing unsuitable conditions to sulphide-sensitive seagrasses [@Holmer2009].

The only high rank taxonomic group showing strong seasonal fluctuations was *Cyanobacteria*. Cyanobacterial sequences were more pronounced in November and December than in spring and summer. In the months of high cyanobacterial sequence abundances the majority of sequences from this group were classified as *Pleurocapsa*, a group known to colonize different living and non-living surfaces [@Longford2007; @Burns2004; @Mobberley2012; @Reisser2014; @Kolda2020]. While we observed a strong temporal pattern for this group, a study of surface sediment cyanobacterial communities did not find any seasonal dynamics for *Pleurocapsa* [@Kolda2020], indicating a possibility that there is a reduced selection of the epiphytic community by the seagrass during periods of low photosynthetic activity [@Zavodnik1998], causing leaves to become a suitable surface for non-specific colonizers. Beside all these groups comprising most of the sequences, a set of taxa present in lower proportions and showing seasonal patterns was identified. This group was comprised of e.g. *Bacteroidota* sequences without known relatives within *Flavobacteriaceae* and *Rhodothermaceae*, the alphaproteobacterial *Amylibacter* and the gammaproteobacterial *Methylotenera*, *Reinekea* and sequences without known relatives within *Cellvibrionaceae* (Figures \ref{bactero} and \ref{gamma}).

It is possible that *Flavobacteriaceae* and *Rhodothermaceae* are occupying similar niches with *Rhodothermaceae* being more adapted to higher temperatures as it is known that culturable members of this family exhibit mesophilic and thermophilic characteristics [@Park2014]. This would explain why we observed a higher presence of *Rhodothermaceae* in the warmer period of the year. A strain belonging to the *Rhodobacteraceae* genus *Amylibacter* has been isolated from the surface of a green macroalga indicating that members of this group can exhibit surface attached life style [@Nedashkovskaya2016]. In addition, since this is a relatively novel genus it is possible that novel taxa within *Rhodobacteraceae* will be described in the future elucidating the taxonomy of the currently high proportion of *Rhodobacteraceae* sequences without known relatives. The genus *Methylotenera* belongs to the methylotrophic family *Methylophilaceae*, a group capable of oxidising non-methane single-carbon compounds such as methanol and methylamine [@Chistoserdova2018]. Interestingly, angiosperms produce methanol during cell-wall synthesis [@Nemecek-Marshall1995; @Dorokhov2018], so it is not surprising that we found members of this genus only on *C. nodosa* and in spring, during a period of maximum seagrass proliferation. Other studies have also found *Methylotenera*-specific sequences associated with seagrass roots and leaves indicating that this group members are important constituents of the seagrass microbiome [@Crump2018; @Sanders-Smith2020]. Genomic and physiological analyses of cultivated *Cellvibrionaceae* and *Reinekea* members showed the capabilities to use important algal polysaccharides [@Xie2017; @Avci2017] indicating their possible involvement into the degradation of *C. cylindracea* polysaccharides and/or the control of its algal epiphytes.

The epiphytic microbial community associated with marine macrophytes is undergoing seasonal changes that can be attributed to the fluctuations of environmental conditions, the growth cycle of macrophytes inhabiting temperate zones or the combined effect of both. In the present study, we could identify in analysed high rank taxa phylogenetic groups present throughout the year, comprising most of the sequences and a lower proportion of taxa showing seasonal patterns connected to the macrophyte growth cycle (Figures \ref{community} and \ref{desulfo}). Studies focusing on functional comparisons between communities associated with different hosts showed that the majority of functions could be found in every community, indicating functional redundancy [@Roth-Schulze2016]. This difference between phylogenetic variability and functional stability has been explained by the lottery hypothesis assuming an initial random colonization step performed by a set of functionally equivalent taxonomic groups [@Burke2011; @Roth-Schulze2016]. It is possible that functional redundancy is a characteristic of high abundance taxa detected to be present throughout the year, while seasonal and/or host-specific functions are an attribute of taxa displaying successional patterns. Further studies connecting taxonomy with functional properties will be required to elucidate the degree of functional redundancy or specificity in epiphytic microbial communities.

\newpage
## Acknowledgments
This work was funded by the Croatian Science Foundation through the MICRO-SEAGRASS project (project number IP-2016-06-7118). ZZ and GJH were supported by the Austrian Science Fund (FWF) project ARTEMIS (project number P28781-B21). We would like to thank Margareta Buterer for technical support, Paolo Paliaga for help during sampling and the University Computing Center of the University of Zagreb for access to the computer cluster Isabella.

\newpage
## References
\interlinepenalty=10000
\setlength{\emergencystretch}{3.7em}
<div id="refs"></div>

\newpage 
\setlength\parindent{0pt}

## Figure legends
**\autoref{map}.** \nameref{map}

**\autoref{pcoa}.** \nameref{pcoa}

**\autoref{shared}.** \nameref{shared}

**\autoref{community}.** \nameref{community}

**\autoref{cyano}.** \nameref{cyano}

**\autoref{bactero}.** \nameref{bactero}

**\autoref{alpha}.** \nameref{alpha}

**\autoref{gamma}.** \nameref{gamma}

**\autoref{desulfo}.** \nameref{desulfo}

\newpage
## Figures
```{r out.width="100%", fig.align="center", fig.cap="Location of the mixed (\\textit{C. nodosa} and \\textit{C. cylindracea}) and monospecific (\\textit{C. cylindracea}) settlement in the Bay of Funtana, northern Adriatic Sea (© OpenStreetMap contributors, www.openstreetmap.org/copyright).\\label{map}", fig.pos="H"}
include_graphics(normalizePath("results/figures/map.jpg"))
```

```{r out.width="55%", fig.align="center", fig.cap="Principal Coordinates Analysis (PCoA) of Bray-Curtis distances based on OTU abundances of bacterial and archaeal communities from the surfaces of the macrophytes \\textit{C. nodosa} (mixed settlement) and \\textit{C. cylindracea} (mixed and monospecific settlement) and in the ambient seawater. Samples from different environments or seasons are labeled in different color and shape. The proportion of explained variation by each axis is shown on the corresponding axis in parentheses.\\label{pcoa}", fig.pos="H"}
include_graphics(normalizePath("results/figures/pcoa_figure.jpg"))
```

```{r out.width="100%", fig.align="center", fig.cap="Proportion of shared bacterial and archaeal communities (Bray-Curtis similarity coefficient) and shared bacterial and archaeal OTUs (Jaccard's similarity coefficient) between consecutive sampling dates and from the surfaces of the macrophytes \\textit{C. nodosa} (mixed settlement) and \\textit{C. cylindracea} (mixed and monospecific settlement) and in the ambient seawater.\\label{shared}", fig.pos="H"}
include_graphics(normalizePath("results/figures/seasonal_shared.jpg"))
```

```{r out.width="85%", fig.align="center", fig.cap="Taxonomic classification and relative contribution of the most abundant ($\\geq$ 1 \\si{\\percent}) bacterial and archaeal sequences on the surfaces of the macrophytes \\textit{C. nodosa} (mixed settlement) and \\textit{C. cylindracea} (mixed and monospecific settlement) and in the ambient seawater. NR -- No Relative (sequences without known relatives within the corresponding group)\\label{community}", fig.pos="H"}
include_graphics(normalizePath("results/figures/community_bar_plot.jpg"))
```

```{r out.width="85%", fig.align="center", fig.cap="Taxonomic classification and relative contribution of the most abundant ($\\geq$ 1 \\si{\\percent}) cyanobacterial sequences on the surfaces of the macrophytes \\textit{C. nodosa} (mixed settlement) and \\textit{C. cylindracea} (mixed and monospecific settlement) and in the ambient seawater. The proportion of cyanobacterial sequences in the total bacterial and archaeal community is given above the corresponding bar. NR -- No Relative (sequences without known relatives within the corresponding group)\\label{cyano}", fig.pos="H"}
include_graphics(normalizePath("results/figures/cyanobacteria_bar_plot.jpg"))
```

```{r out.width="85%", fig.align="center", fig.cap="Taxonomic classification and relative contribution of the most abundant ($\\geq$ 2 \\si{\\percent}) sequences within the \\textit{Bacteroidota} on the surfaces of the macrophytes \\textit{C. nodosa} (mixed settlement) and \\textit{C. cylindracea} (mixed and monospecific settlement) and in the ambient seawater. The proportion of sequences classified as \\textit{Bacteroidota} in the total bacterial and archaeal community is given above the corresponding bar. NR -- No Relative (sequences without known relatives within the corresponding group)\\label{bactero}", fig.pos="H"}
include_graphics(normalizePath("results/figures/bacteroidota_bar_plot.jpg"))
```

```{r out.width="85%", fig.align="center", fig.cap="Taxonomic classification and relative contribution of the most abundant ($\\geq$ 2 \\si{\\percent}) alphaproteobacterial sequences on the surfaces of the macrophytes \\textit{C. nodosa} (mixed settlement) and \\textit{C. cylindracea} (mixed and monospecific settlement) and in the ambient seawater. The proportion of alphaproteobacterial sequences in the total bacterial and archaeal community is given above the corresponding bar. NR -- No Relative (sequences without known relatives within the corresponding group)\\label{alpha}", fig.pos="H"}
include_graphics(normalizePath("results/figures/alphaproteobacteria_bar_plot.jpg"))
```

```{r out.width="85%", fig.align="center", fig.cap="Taxonomic classification and relative contribution of the most abundant ($\\geq$ 1 \\si{\\percent}) gammaproteobacterial sequences on the surfaces of the macrophytes \\textit{C. nodosa} (mixed settlement) and \\textit{C. cylindracea} (mixed and monospecific settlement) and in the ambient seawater. The proportion of gammaproteobacterial sequences in the total bacterial and archaeal community is given above the corresponding bar. NR -- No Relative (sequences without known relatives within the corresponding group)\\label{gamma}", fig.pos="H"}
include_graphics(normalizePath("results/figures/gammaproteobacteria_bar_plot.jpg"))

```

```{r out.width="85%", fig.align="center", fig.cap="Taxonomic classification and relative contribution of the most abundant ($\\geq$ 1 \\si{\\percent}) sequences within the \\textit{Desulfobacterota} on the surfaces of the macrophytes \\textit{C. nodosa} (mixed settlement) and \\textit{C. cylindracea} (mixed and monospecific settlement) and in the ambient seawater. The proportion of sequences classified as \\textit{Desulfobacterota} in the total bacterial and archaeal community is given above the corresponding bar. NR -- No Relative (sequences without known relatives within the corresponding group)\\label{desulfo}", fig.pos="H"}
include_graphics(normalizePath("results/figures/desulfobacterota_bar_plot.jpg"))
```
